### ISSUE TITLE
we need a parameter to change the pivot of the tile

### ISSUE THREAD

**Issue by sharebophar** [bjorn/tiled#2761]
we need a parameter to change the pivot of the tile,default position is the center of the tile

**Comment by bjorn** [comment]
Actually the default pivot of a tile object is its bottom-left corner. It's just that the object selection tool always rotates the selected objects around their collective center, unless you first move the rotation origin (you can drag it around, though it currently resets to the center too often).

To change the pivot of the tile objects a new "Tile Object Alignment" tileset property was recently added, which is available in the current snapshots (to be released as Tiled 1.4 in a few weeks). You can set it to center to have the rotation of the object applied around the center of its image. Note that this still will not change the behavior of the object selection tool.

It would probably be helpful to add a different rotation handle to make it easier to rotate objects around their pivot. For now you can do this by editing the Rotation property directly in the Properties view.

**Comment by bjorn** [comment]
@kennex09 There's a long requested feature of per-tile pivot points (see also #871), which I do plan to get around to at some point. Or are you requesting a pivot point _per object_?

**Comment by kennex09** [comment]
Yes, a pivot point per object would be ideal.  Specifically for rotation.  Since not all objects need rotated.

**Comment by bjorn** [comment]
@kennex09 Hmm, well for objects that don't need to be rotated the pivot point is irrelevant. What use-case do you have to have different pivot points per object, rather than per tile?

**Comment by kennex09** [comment]
I was use image objects for sprite generation. When one of those objects needs rotated, the pivot point is not by the center so the x and y shows in a different place on Tiled than when it's just run in a program without any additional changes.  If that makes sense.
